raft/src/main/java/org/apache/kafka/raft/FileBasedStateStore.java

```diff
@@ -91,14 +92,17 @@ private QuorumStateData readStateFromFile(File file) throws IOException {
 
             final short dataVersion = dataVersionNode.shortValue();
             return QuorumStateDataJsonConverter.read(dataObject, dataVersion);
+        } catch (IOException e) {
+            throw new UncheckedIOException(
+                    String.format("Error while reading the Quorum status from the file %s",file),e);
```

Comment:
there should be a space after a comma:
```
String.format("Error while reading the Quorum status from the file %s", file), e);
```

Also, the indent is not correct here. Please fix it. And same to other places.
Thanks.

Reply:
Sorry, the code format problem, will pay attention later, this time rushed submission